RPIPs/RPIP-49.md

```diff
@@ -100,32 +100,27 @@ This RPIP introduces a delay after protocol upgrades have been confirmed but pri
 The tokenomics rework package will likely be split between two protocol upgrades: Saturn 1 and Saturn 2.
 
 ### Saturn 1
-
-* [RPIP-43: Megapools](RPIP-43.md)
-  * Including ETH-only validators.
-* [RPIP-42: Bond curves](RPIP-42.md)
-  * Framework
-  * 4ETH minimum bond
-* [RPIP-59: Deposit Mechanics](RPIP-59.md)
-  * Standard and express queues.
-* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Splits between rETH, node operators, and vote-eligible staked RPL staked in megapools
-  * RPL issuance rewards no longer have a minimum stake required
-* RPL Value Capture - Increased share to voting Node Operators.
+* [RPIP-42: Bond curves](RPIP-42.md) - Partial Inclusion
+  * 4 ETH minimum bond
```

Comment:
```suggestion
  * 4 ETH bonds only
```

Reply:
Or "4 ETH bonds for all validators"?

Reply:
Technically it's 4-8-4-*? So not sure I can say that.

Reply:
I like "4 ETH bond per validator"

Reply:
> What do you mean? Each new validator will require an additional 4 ETH bond. The array will go 4-8-12-16-*, but each index is how much capital you need for the next validator (4th validator requires 16 ETH or 4ETH bond per validator)?
> 
> Maybe "4 ETH bond per validator"?

Possible I've been misunderstanding this for ages lol. Is it not:
4 ETH for first.
12 ETH for second.
16 ETH for third.
?

I thought the array value was the delta that you needed, not the total?

Reply:
Will have to think it out tomorrow.

Reply:
It is total amount you need, so Saturn 1 is flat 4ETH bonds
Total per index: [4,8,12,…]
Alternatively just the “extra cost per index: [4,4,4,…]

Saturn changes to:
[4,8,9.5,11,13.5…]
or
(Additional cost): [4,4,1.5,1.5,1.5,…]

Reply:
Confirming samus has it right